hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsDfsClient.java

```diff
@@ -321,6 +320,7 @@ public AbfsRestOperation listPath(final String relativePath,
   /**
    * Get Rest Operation for API <a href = https://learn.microsoft.com/en-us/rest/api/storageservices/datalakestoragegen2/path/create></a>.
    * Create a path (file or directory) in the current filesystem.
+   *
```

Comment:
No changes in this file, additional changes can be removed

Reply:
taken.